sql/amo_dev/amo_stats_installs_v3/query.sql

```diff
@@ -0,0 +1,22 @@
+SELECT
+  *
+FROM
+  amo_prod.amo_stats_installs_v3
+WHERE
+  submission_date = @submission_date
+  AND (
+    hashed_addon_id IN (
+      '{db55bb9b-0d9f-407f-9b65-da9dd29c8d32}', -- :willdurand
```

Comment:
`hashed_addon_id` is a problem for the dev table. We either need the list of hashed IDs to insert here, or we need to be able to join back to unhashed IDs before doing this filtering.

Reply:
We could hash those add-on IDs if needed.